### DISCUSSION TITLE
reinstalled Xcom 2 on another drive, mods all duplicated.

### DISCUSSION THREAD

**Discussion by ShinyNobody** [X2CommunityCore/xcom2-launcher#327] [Help]
Hello,
like i said all mods are duplicated. the new downloaded mods are also not sorted. I did not find anything online how to fix it. i have my original settings.jason but that does not help. i changed the location in this file after reinstalling xcom. Do i have to manually change everything or is there a trick? deleting all mods manually and activating only the ones i nead for my playtrough is is alot of hassle with over 600 mods.

**Comment by Dragon-32** [comment]
I think I used Search and Replace in my settings.json when I upgraded my PC. and copied my install over.

**Reply by ShinyNobody** [reply]
How can i try t? i cannot find anything called like that in my settings.json

**Reply by Dragon-32** [reply]
Open the file in a text editor (I use Notepad++) and use its search and replace function, take note that "folder1/folder2" is ""folder1//folder2" in the .json
